zqd/handlers_zeek_test.go

```diff
@@ -229,6 +229,7 @@ func (r *packetPostResult) postPcap(t *testing.T, file string) {
 	if r.err == nil {
 		r.readPayloads(t, stream)
 	}
+
```

Comment:
Was this on purpose?

Reply:
nope, leftover from debugging, will remove